### ISSUE TITLE
Tool Says Jit is Done But My Dolphin IOS Freezes And When it unfreezes jit is not usable

### ISSUE THREAD

**Issue by Irex126** [fritzlb/iOS17-JIT-WIN#70]
The tool Works normally as intended just that when it before it says done dolphin freezes and never unfreezes unless i close the app and try after 5 mins and in the occasions it does unfreeze after a min it still tells me waiting for jit how can i fix this i really wanna use dolphin alot

**Comment by fritzlb** [comment]
This sounds a little like lldb never deattaches properly. Could you provide the full terminal output in debug mode? That should make it easier to debug

**Comment by Irex126** [comment]
Yeah im kind of dumb so could you explain how to get the full terminal output? Im so sorry

**Comment by aiden04** [comment]
I'm experiencing the same issue. Used Sidestore to sideload DolphinIOS stable, running JIT-WIN appears to work fine looking at console, but app freezes after starting and waiting around 5 mins crashes the app. The app seems to freeze shortly after the console says `Running debug commands. This might take a few minutes`. Running IOS 17.5 on an iPhone SE (2020).

JIT-WIN Console Output:
```
C:\Users\####\OneDrive\Desktop\iOS17-JIT-WIN-main>jit_enabler_better.py me.oatmealdome.dolphinios-njb.DSG59N9K7B
Getting bundle ID...
Got bundle ID: me.oatmealdome.dolphinios-njb.DSG59N9K7B
starting tunnel to device...
This might take a while. In case it freezes, either close this window and kill every python process in task manager or simply reboot your PC.
Sucessfully created tunnel: fd24:bfee:8f9::1 50308
Manually trying to mount DeveloperDiskImage (this seems to prevent errors on some systems)...
Diskimage already mounted.
Starting app...
Started app. PID: 1288
Starting debug server...
Started debug server with connection details: [fd24:bfee:8f9::1]:51680
Run debugging commands...
This might take a few minutes.
done.
```
I tried to find a log for DolphinIOS but was unable to find any.

**Comment by Withermask** [comment]
im having the same problem but with pojav launcher (ios 17.4.1 iphone 13 pro)

**Comment by fritzlb** [comment]
A little time ago this only happened if the app has been sideloaded using a method that's not supported, but altstore and sidestore should be fine. 
Tbh I really don't know what causes this right now, also I can't reproduce it

**Comment by Irex126** [comment]
Thats fine but could you atleast look at my other issue the most recent one?

**Comment by Withermask** [comment]
Oh.  Bb o ooo p ok 🌊lol🌊.    Of
